layouts/legal/single.html

```diff
@@ -5,7 +5,7 @@
                 <aside>
                     <ul class="md:w-8/12 p-0 list-none">
                         {{ range .Site.Menus.legal }}
-                            <li><a href="{{ .URL }}">{{ .Title }}</a></li>
+                            <li><a data-track="{{ .Title | urlize }}" href="{{ .URL }}">{{ .Title }}</a></li>
```

Comment:
Why track these with a title and not a category like the others (since the href will make it clear what the destination is)?